src/client/content_cao.h

```diff
@@ -180,6 +177,12 @@ class GenericCAO : public ClientActiveObject
 		return m_matrixnode->getRelativeTransformationMatrix();
 	}
 
+	inline const core::matrix4 &getAbsolutePosRotMatrix()
```

Comment:
Would anything speak against this? (I'm a newbie in cv-qualified member functions.)
```suggestion
	inline const core::matrix4 &getAbsolutePosRotMatrix() const
```
([`getAbsoluteTransformation`](http://irrlicht.sourceforge.net/docu/classirr_1_1scene_1_1_i_scene_node.html#a9c46ea82efd7ae30437fb7c385809050) is const.)

Reply:
This method simply returns `irr::scene::IDummyTransformationSceneNode::getAbsoluteTransformation` which is also a `const` method. So... why not? :D

Reply:
Done